examples/google-secure-signals-integration/with_sdk_v3/views/intro.html

```diff
@@ -1,13 +1,11 @@
-<h1>Server-Side Integration Example, UID2 JavaScript SDK</h1>
+<h1>Example for Client-Server UID2 SDK Integration with Google Secure Signals</h1>
 <p class="intro">
   This example demonstrates how a content publisher can use the UID2 services and the
-  <a href="https://unifiedid.com/docs/sdks/sdk-ref-javascript"
-    >UID2 SDK for JavaScript</a
-  >
+  <a href="https://unifiedid.com/docs/sdks/sdk-ref-javascript">UID2 SDK for JavaScript</a>
   to implement the
   <a
-    href="https://unifiedid.com/docs/guides/integration-google-ss#server-side-integration"
-    >server-side UID2 integration</a
+    href="https://unifiedid.com/docs/guides/integration-google-ss#sdk-for-javascript-client-side-integration"
+    >client-side UID2 integration</a
```

Comment:
Just checking - these changes have client-server and client-side - is this expected?

Reply:
It's a good question.  I think so.  The link calls out the client-side integration example but the overall example is client-server since the server is generating the token but the client is initializing the sdk with the identity from the server.

Reply:
i dont think this is correct - 

short answer:
it should be referring to Client-Server Integration and better to link to this instead as a quick fix:
https://unifiedid.com/docs/guides/integration-javascript-client-server

long answer:
We should add a Client-Server Integration option section in [GAM Secure Signals](https://unifiedid.com/docs/guides/integration-google-ss) instead of linking to [SDK for JavaScript Client-Side Integration](https://unifiedid.com/docs/guides/integration-google-ss#sdk-for-javascript-client-side-integration)

So in GAM Secure Signal page, we should mention 3 integration methods - Server-Side, Client-Server and Client-Side. cc @genwhittTTD on this but probably it is best to talk thru between Gen/Scott/Me after new year.